README.md

```diff
@@ -1,64 +1,228 @@
+  discovery-redis.service         loaded active running Redis for Discovery
+  discovery-server.service        loaded active running Discovery Server
+
+LOAD   = Reflects whether the unit definition was properly loaded.
+ACTIVE = The high-level unit activation state, i.e. generalization of SUB.
+SUB    = The low-level unit activation state, values depend on unit type.
+6 loaded units listed. Pass --all to see loaded but inactive units, too.
+To show all installed unit files use 'systemctl list-unit-files'.
 ```
 
-Helper shell functions:
+Starting and stopping Discovery is managed via the **discovery-app.service** service. It declares its dependencies to the other quadlets and is responsible for the Discovery quadlet orchestration.
+
+### Discovery Quadlet Orchestration
+
+- **discovery-app.service**
+  - Depends on and automatically starts and stops:
+      - discovery-server
+      - discovery-celery-worker
+      - discovery-redis
+      - discovery-db
+  - Starts After:
+      - discovery-server
+
+- **discovery-celery-worker.service**
+  - Starts After:
+      - discovery-server
+
+- **discovery-server.service**
+  - Starts After:
+      - discovery-db
+      - discovery-redis
+ 
+- **discovery-redis.service**
+
+- **discovery-db.service**
+
+
+
+## Starting Discovery
+
+Starting Discovery and all related quadlets is done via the _discovery-app_ quadlet as follows:
 
 ```
-$ list-dsc            - List Discovery systemd units
-$ list-generated-dsc  - List Discovery generated systemd services
-$ start-dsc           - Start a Discovery service
-$ stop-dsc            - Stop a Discovery service
-$ restart-dsc         - Stops then re-Starts a Discovery service
+discovery $ systemctl --user start discovery-app
+discovery $
 ```
 
 
+## Accessing Discovery
+
+You can access the Discovery console on the Machine hosting the Discovery Quadlet Services by opening up a browser and accessing port 9443.
 
-## Additional research:
+In this example, the host name is aab-dsc, so simply access the following URL:
 
-- [ ] Control service cpu/mem/net limits via cgroups (Control Groups)
+```
+https://aab-dsc:9443
+```
+
+
+## Stopping Discovery
 
-- [ ] How customers can specify alternate image repositories
+Stopping Discovery and all related quadlets is done as follows:
 
-- [ ] How is upgrade handled
+```
+discovery $ systemctl --user stop discovery-app
+discovery $
+```
 
+## Uninstalling Discovery
 
-## References:
+For uninstalling Discovery and all related files and services, run the following command:
 
-- https://www.redhat.com/sysadmin/multi-container-application-podman-quadlet
-- https://www.redhat.com/sysadmin/quadlet-podman
-- https://github.com/containers/podman/blob/main/docs/source/markdown/podman-systemd.unit.5.md
-- https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html
-- https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html
-- https://github.com/containers/quadlet/tree/main/docs
```

Comment:
I'd leave this reference section. It is very handy! Or did you move it to somewhere else and I missed it?

Reply:
was useful during development, but yea, I'll bring it back in 👍